sound/soc/sof/intel/hda-dai.c

```diff
@@ -414,6 +414,49 @@ static struct snd_soc_cdai_ops sof_probe_compr_ops = {
 #endif
 #endif
 
+static int ssp_dai_hw_params(struct snd_pcm_substream *substream,
```

Comment:
comment on teh commit message. This isnt adding the BE fixup function but rather the BE hw_params callback right?

Reply:
and we should also mention in the commit message that this patch would introduce a new IPC that doesnt exist today.

Reply:
yes will fix